download.md

```diff
@@ -11,9 +11,9 @@ redirect_from:
 # Downloads
 
 <section id="downloads">
-      <a href="{{ site.stable_url }}"><span>Stable<br/>0.9.3</span></a>
-      <a href="https://github.com/kimai/kimai/zipball/master"><span>DEV<br/>0.9.4</span></a>
-      <a href="https://github.com/kimai/manuals/raw/master/documentation.pdf"><span>Docu<br/>PDF</span></a>
+  <a href="{{ site.stable_url }}"><span>Stable<br/>0.9.3</span></a>
+  <a href="https://github.com/kimai/kimai/archive/master.zip"><span>DEV<br/>0.9.4</span></a>
```

Comment:
Is there a difference beside the filename? Both links spit out the version with the latest changes in INSTALL.md